models.json

```diff
@@ -381,5 +381,69 @@
         }
       }
     }
+  },
+  "Middleware": {
+    "properties": {
+      "id": {"type": "string", "id": true, "json": false},
+      "name": {"type": "string", "json": false},
+      "phase": {"type": "string", "json": false},
```

Comment:
I think that `MiddlewarePhase` relation "hasMany phases" will add a property `Middleware.phaseId`. Is it a good idea to have both `phase` and `phaseId`? Should the model ensure that these two properties are always consistent, i.e. prevent a situation where `phase` points to a different entity than `phaseId`?